app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Telling candidates they need to do a subject knowledge enhancement course
+description: How we designed a way to tell candidates they have a subject knowledge enhancement course as a condition of their offer.
+date: 2023-03-17
+tags:
+  - making an offer
+  - application details
+ogImage:
+  src: /apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/ske-cover-image.png
+  alt: "Illustration with the text 'Offer received' and inforation about doing an 8 week subject knowledge enhancement course in physics"
+
+---
+![Illustration with the text 'Offer received' and inforation about doing an 8 week subject knowledge enhancement course in physics](ske-cover-image.png)
+
+Previously, we designed a way to help training providers add a subject knowledge enhancement (SKE) course as a condition of an offer.
+
+When this offer comes through to the candidate, we need a way for them to see information about the SKE course.
+
+We tested these designs and improved the content shown to candidates. We released this new feature in March 2023.
+
+## The issue
+
+Subject knowledge enhancement (SKE) courses are currently available in 9 secondary subjects:
+
+* biology
+* chemistry
+* computing
+* design and technology
+* English
+* languages
+* mathematics
+* physics
+* religious education
+
+Providers can ask candidates to do an SKE course if they need to improve their subject knowledge before they start their initial teacher training (ITT).
+
+From previous research, candidates often do not know what an SKE course is when they're asked to do one.
+
+Some of the main issues candidates have are not knowing:
+
+* why they have to do an SKE course
+* how to find an SKE course or who to do it with (SKE providers and ITT providers can be different)
+* SKE courses are free
+* they can get a bursary of £175 a week while they do their SKE course
+* how they can study their course (for example, online, in-person or both)
+
+## What we did
+
+After the training provider sends an offer to a candidate with an SKE condition, we show some guidance before a candidate accepts their offer. This helps explain what an SKE is, so the candidate can make an informed decision about accepting the offer or not.
```

Comment:
Should we always use "SKE course" and not just "SKE"? (Annoying though it is...)

```suggestion
After the training provider sends an offer to a candidate with an SKE course condition, we show some guidance before a candidate accepts their offer. This helps explain what an SKE course is, so the candidate can make an informed decision about accepting the offer or not.
```